base/src/main/proto/spine/net/url.proto

```diff
@@ -9,87 +9,99 @@ option java_multiple_files = true;
 option java_package = "io.spine.net";
 option java_outer_classname = "UrlProto";
 
-option (experimental_all) = true;
-
-// Protobuf definition for URL.
-//
-// Can be defined in raw string form or as a structurized parsed record. Validity of URL is not
-// guaranteed, if the raw form is used.
+// A Universal Resource Locator.
 //
 message Url {
-    oneof value {
-        string raw = 1;
-        Record record = 2;
-    }
 
-    // Structured record.
+    reserved 1 to 2;
+    reserved "raw";
+    reserved "record";
+
+    // The value of the URL.
+    //
+    // Validity of the value should be provided by the code creating instances of this message.
     //
-    // This type represents URL according to RFC 3986.
-    // Using this type makes it easier to operate concrete URL attributes.
-    // Please see https://tools.ietf.org/html/rfc3986.
-    message Record {
-        // Application layer protocol
-        Protocol protocol = 1;
+    string spec = 3 [(required) = true];
```

Comment:
Why `spec` and not `value`?

Reply:
`value` used to be a `oneof` switch. I'll add it to reserved too.